com.claudiocambra.brickbuster.json

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "com.claudiocambra.brickbuster",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "brickbuster_linux.x86_64",
+    "modules": [
+        {
+            "name": "BrickBuster",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D brickbuster_linux.x86_64 /app/bin/brickbuster_linux.x86_64"
```

Comment:
Ideally it should be built from source so all architectures are supported (currently aarch64). It's not a blocker though. Remember to limit builds to x86_64: https://github.com/flathub/flathub/wiki/App-Maintenance#limiting-the-set-of-architectures-to-build-on

Reply:
Unfortunately due to how complicated it becomes to compile and export a Godot project like BrickBuster from source, I think the binary is probably the best approach for now (especially since the engine currently doesn't provide Linux exports for anything other than x86_64 anyway).

Per your advice I've limited builds to x86_64 :)